client/src/components/ProfileTabs/style.css

```diff
@@ -8,4 +8,12 @@
     display: flex;
     justify-content: center;
     align-items: center;
+}
+.profile-tabs-holder {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
+.ant-tabs {
+    width: 70%;
```

Comment:
For this to not affect other styles, so add a class for your container and add these styles to these classes inside your container class.